packages/commonwealth/client/scripts/views/pages/CommunityGroupsAndMembers/common/constants.ts

```diff
@@ -48,9 +51,16 @@ export const conditionTypes = [
 // Get chain id's from the app.config.chains for all eth and cosmos chains
 export const chainTypes = app.config.nodes
   .getAll()
-  .filter((chain) => chain.ethChainId || chain.cosmosChainId)
+  .filter(
+    (chain) =>
+      chain.ethChainId || chain.cosmosChainId || chain.balanceType === 'solana',
+  )
   .map((chain) => ({
-    chainBase: chain.ethChainId ? 'ethereum' : 'cosmos',
-    value: chain.ethChainId || chain.cosmosChainId,
+    chainBase: chain.ethChainId
+      ? 'ethereum'
+      : chain.balanceType === 'solana'
+      ? 'solana'
+      : 'cosmos',
+    value: chain.ethChainId || chain.cosmosChainId || 0,
```

Comment:
don't we have some identifier for solana chains? I am worry that `0` might not be safe in this case

Reply:
Solana doesnt technically have a chain id because it is its own eco-system. thought about make evm_chain_id nullable but I feel that could cause more issues. The idea here is that a combination of type SPL with evm_chain_id 0 default will always route to solana logic which ignore the evm_chain_id. Thoughts?

Reply:
I thought that there might be some issues with selecting a chain in create community flow but for solana the dropdown is disabled because we always targeting solana mainnet. In this case I guess everything is good 

![image](https://github.com/hicommonwealth/commonwealth/assets/14819225/d9bfc7e4-d931-4724-922c-acc4b4371bbe)